Find a functionally equivalent code:android.content.res.TypedArray.getResources:COMMENT
Method Modifier: public      
Comment:/**
 * Return the Resources object this array was loaded from.
 */

@@ -1,3 +1,6 @@
 {
+    if (mRecycled) {
+        throw new RuntimeException("Cannot make calls to a recycled instance!");
+    }
     return mResources;
 }

